subprojects/antlr/src/integTest/groovy/org/gradle/api/plugins/antlr/IncrementalAntlrTaskIntegrationTest.groovy

```diff
@@ -117,12 +117,12 @@ class IncrementalAntlrTaskIntegrationTest extends AbstractAntlrIntegrationTest {
 
         then:
         succeeds("generateGrammarSource")
-        test1TokenFile.assertHasChangedSince(test1TokensFileSnapshot);
-        test1LexerFile.assertHasChangedSince(test1LexerFileSnapshot);
-        test1ParserFile.assertHasChangedSince(test1ParserFileSnapshot);
+        test1TokenFile.assertHasChangedSince(test1TokensFileSnapshot)
+        test1LexerFile.assertHasChangedSince(test1LexerFileSnapshot)
+        test1ParserFile.assertHasChangedSince(test1ParserFileSnapshot)
     }
 
-    def "output for removed grammar file is not handled correctly"() {
+    def "output for removed grammar file is handled correctly"() {
```

Comment:
\o/

Reply:
Only the test name was wrong, no test was fixed...